Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/presencetracing/organizer/warn/info/TraceLocationWarnInfoViewModel.kt

```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.ui.presencetracing.organizer.warn.info
```

Comment:
Is this necessary? We have a static fragment with one button only for navigation to the next fragment